docs/getting-started-tutorial.md

```
@@ -0,0 +1,412 @@
+// One agent can interact with many others
+const user = Agent('User', { name: 'Alice' });
+const validator = Agent('Validator', { valid: false });
+const logger = Agent('Logger', { entries: [] });
+
+const validateRule = ActionRule(user.ports.main, validator.ports.main, /* ... */);
+const logRule = ActionRule(user.ports.main, logger.ports.main, /* ... */);
+```
+
+## Troubleshooting
+
+### Common Issues:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)